modules/aleph/src/Aleph/Entity/AlephRequestResponse.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+namespace Drupal\aleph\Aleph\Entity;
+
+/**
+ * @file
+ * Provides the AlephRequestResponse.
+ */
+
+/**
+ * Class AlephRequestResponse
+ *
+ * This entity describes the response from Aleph.
+ *
+ * @package Drupal\aleph\Aleph\Entity
+ */
+class AlephRequestResponse {
+
+  /**
+   * @var $replyCode
+   *  The reply code is 0000 on success.
```

Comment:
It is probably worth showing that the reply code is a string - not a integer.

If we have a link to a description of these reply codes and other aspects of the aleph response then adding it would be appropriate.